twiggy/tests/all/expectations/garbage

```diff
@@ -1,12 +1,9 @@
  Bytes │ Size % │ Garbage Item
 ───────┼────────┼────────────────────────────────
-    11 ┊  5.58% ┊ unusedAddThreeNumbers
-     8 ┊  4.06% ┊ unusedAddOne
+    12 ┊  6.09% ┊ unusedAddThreeNumbers
+     9 ┊  4.57% ┊ unusedAddOne
```

Comment:
In general, most of the test changes look like this: A single byte being added to the node that now represents both the code _and_ function entries. Other changes, such as the rest of this file, are because there are fewer items listed in the 'total rows' message.